numpy/lib/stride_tricks.py

```diff
@@ -244,7 +244,7 @@ def broadcast_arrays(*args, **kwargs):
     subok = kwargs.pop('subok', False)
     if kwargs:
         raise TypeError('broadcast_arrays() got an unexpected keyword '
-                        'argument {}'.format(kwargs.pop()))
+                        'argument {!r}'.format(kwargs.keys()[0]))
```

Comment:
`!r` (= repr) will put quotes around the key and escape any weird junk that might be in it, both of which seem like good things to me in this context